use-cases/bed-allocation/overconstrained01.txt

```diff
@@ -0,0 +1,898 @@
+ARTICLE BENCHMARK DATA SET
```

Comment:
Why is this file here, and should it be here?
No other quickstart carries its own data files.

Reply:
No, it should not. Let me remove it.